internal/app/export/service.go

```diff
@@ -17,15 +16,20 @@ import (
 	"github.com/mainflux/export/internal/pkg/routes"
 	"github.com/mainflux/export/internal/pkg/routes/mfx"
 	"github.com/mainflux/export/pkg/config"
+	"github.com/mainflux/mainflux/errors"
 	logger "github.com/mainflux/mainflux/logger"
 	nats "github.com/nats-io/nats.go"
 )
 
+var (
+	errNoCacheConfigured = errors.New("No cache configured")
+)
+
 type Service interface {
 	Start(queue string) error
 	Subscribe(nc *nats.Conn)
 	Logger() logger.Logger
-	Publish(subject, topic string, payload []byte)
+	Publish(subject, topic string, payload []byte) errors.Error
```

Comment:
Maybe embedded Publisher would be more elegant. Detail ;)

Reply:
@mteodor Since we have `Publisher` interface that matches this signature, why not? Like:
```go
type Service interface {
    publish.Publisher
    Start(queue string) error // or errors.Error?
...
}
```

Speaking of which, `publish.Publisher` sounds obscure. Maybe this interface should be declared elsewhere (root)?